README.md

```diff
@@ -261,6 +261,24 @@ If you want to build a specific module later on, you can do the same but with ju
 module you wanna build. It'll get added to the same directory where the others are.
 Or, if you are particularly lazy, just add the module name to the make invocation.
 
+## FISh
+The `sled.fish` FiFo supports the following commands:
+
+| Command | Action |
+| --- | --- |
+| `/blank` | Blank the output |
+| `/error42` | Exit the program |
+| `/next` | Jump to the next effect |
+| `/then <modulename> [parameters]` | Schedule the given module or command with optional parameters |
+
+Examples:
+```bash
+echo '/next' > sled.fish
+echo '/then /blank' > sled.fish
```

Comment:
Make clear that '/then autoterminal /next' is not possible and '/then autoterminal\n/next' is neccessary

Reply:
Funnyly it crashes when using `echo -e "/then text Ipsum\n/next" > sled.fish`...

Reply:
(at least that got fixed in #108 (text: Fix use after free in text_free)